npm/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "loam-cli",
-  "version": "0.13.2",
+  "version": "0.14.0",
```

Comment:
I added a ticket to make sure to automate this in the future: https://github.com/loambuild/loam/issues/144